projects/demo/src/modules/components/input-phone-international/index.ts

```diff
@@ -25,7 +28,12 @@ import {InheritedDocumentation} from '../abstract/inherited-documentation';
     ],
     templateUrl: './index.html',
     changeDetection,
-    providers: [tuiProvide(ABSTRACT_PROPS_ACCESSOR, PageComponent)],
+    providers: [
+        tuiProvide(ABSTRACT_PROPS_ACCESSOR, PageComponent),
+        tuiInputPhoneInternationalOptionsProvider({
```

Comment:
Update also this line

https://github.com/taiga-family/taiga-ui/blob/552f202bdbbfebaba4d5ff62b0c71630918f7986/projects/demo-cypress/src/tests/input-phone-international.cy.ts#L23